dist/golang/protocol/fees.go

```diff
@@ -458,7 +451,7 @@ func EstimatedTransferResponse(requestTx *wire.MsgTx, dustLimit uint64, feeRate
 			}
 
 			if asset.AssetType == "BSV" {
-				fundingForBSV[i] += totalSent
+				fundingForBSV[masterContractIndex] += totalSent
```

Comment:
I believe this will duplicate the bitcoin amount because it is already being added on line 401

Reply:
The duplication has been removed.